### DISCUSSION TITLE
do you have a solver for solving system of odes in this paper?

### DISCUSSION THREAD

**Discussion by ehtisham409** [lululxvi/deepxde#671] [Q&A] [answered]
I. E. Lagaris, A. Likas, and D. I. Fotiadis, Articial neural networks for solving ordinary and
partial dierential equations, IEEE Transactions on Neural Networks, 9 (1998), pp. 987{
1000.

**Comment by lululxvi** [comment, marked as answer]
Yes. There are examples of ODE system, see ode_system.py for forward problem, and Lorenz_inverse.py for inverse problem.

**Comment by ehtisham409** [comment]
hello sir
do you have documention for deepxde. this will be helpful for
better understanding of code for fresh students in deeplearning.

Your Truely

Ehtisham Karim

**Comment by lululxvi** [comment]
Hi Ehtisham,

Currently I don't have very detailed documentation yet. But the papers/slides listed in the README and the API information https://deepxde.readthedocs.io/en/latest/modules/modules.html should be useful.
